### PR TITLE
PPTP-1184 | content changes for liability weight questions

### PR DESCRIPTION
**Pull request by liampan**
### Description of Work carried through

Content changes.

#### Check list 
 - [ ] `./precheck` was executed (Integration/Component/Unit tests)
 - [ ] `sbt scalafmt test:scalafmt` was executed
 - [ ] Required Environment Config has been amended/added
 - [ ] Links to dependencies have been included (BE/FE work)
 - [ ] User Acceptance Tests (UAT) were run locally and have passed.
 - [ ] No Accessibility errors/warnings reported by Wave

### CODE REVIEW

File: test/uk/gov/hmrc/plasticpackagingtax/registration/views/liability/LiabilityExpectedWeightViewSpec.scala [hunk 1]
```diff
@@ -74,6 +74,12 @@ class LiabilityExpectedWeightViewSpec extends UnitViewSpec with Matchers {
       )
     }
 
+    "display liability weight info" in {
+      view.getElementsByClass("govuk-body").text().text() must include(
```
Comment (RyanKellyHMRC) [inline test/uk/gov/hmrc/plasticpackagingtax/registration/views/liability/LiabilityExpectedWeightViewSpec.scala:78]: Did you really need the double `text().text()` calls?

**Review by RyanKellyHMRC — APPROVED** [review]
One small typo.

File: test/uk/gov/hmrc/plasticpackagingtax/registration/views/liability/LiabilityExpectedWeightViewSpec.scala (hunk 1 above)
Comment (liampan) [reply test/uk/gov/hmrc/plasticpackagingtax/registration/views/liability/LiabilityExpectedWeightViewSpec.scala:78]: oop good spot! Im surpirsed that even compiles

Comment (liampan) [reply test/uk/gov/hmrc/plasticpackagingtax/registration/views/liability/LiabilityExpectedWeightViewSpec.scala]: removed. :)

**Pull request merged**